packages/react/src/components/DetailsList/DetailsList.test.tsx

```diff
@@ -1061,4 +1061,56 @@ describe('DetailsList', () => {
     const groupNameId = checkbox.getAttribute('aria-labelledby')?.split(' ')[1];
     expect(container.querySelector(`#${groupNameId} span`)!.textContent).toEqual('Group 0');
   });
+
```

Comment:
This does not have a group footer rendering.  Will add if happy with GroupFooter.

May also be worth having a group with no child groups....